### ISSUE TITLE
Crash when leaving a lore structure

### ISSUE THREAD

**Issue by sakari119** [anegostudios/VintageStory-Issues#4662]
### Game Version

v1.20 rc1

### Platform

Windows

### Modded

Modded

### SP/MP

Singleplayer

### Description

I was exploring a lore structure and when I left it the game crashed.

### How to reproduce

Enter lore structure
???
Leave
???
Crash

### Logs

```Running on 64 bit Windows 10.0.19045.0 with 64663 MB RAM
Game Version: v1.20.0-rc.1 (Unstable)
11/12/2024 9:29:13 PM: Critical error occurred
Loaded Mods: atwatersedge@1.5.0, blacksmithname@1.2.0, Bluesky@1.1.0, combatoverhaul@0.0.91, decor@1.1.9, DrystoneParity@1.0.0, femininity@0.1.3, hideplayername@2.1.1, immersivecorpsedrop@1.0.2, knappingmastery@1.0.1, mycodiversity@1.0.2, nailsmold@1.0.2, primitivesurvival@3.7.3, RelightTorches@1.0.0, rustandrot@1.0.6, temporal_gears_stack@1.0.0, game@1.20.0-rc.1, vsimgui@1.1.7, AgeOfConfession@2.0.7, armory@0.0.26, betterentityinteraction@1.0.3, betterfirepit@1.1.4, betterhewnfencegates@1.1.0, betterruins@0.3.9, butchering@1.7.3, carryon@1.8.0-pre.1, charcoalpitdoor@1.0.0, commonlib@2.6.1, configlib@1.3.17, danatweaks@3.3.5, divisionoflabor@1.0.8, earlychiseling@1.0.1, foodshelves@1.4.3, gourmand@1.2.7, hardcorewater@1.2.0, herbarium@1.3.0, hydrateordiedrate@1.5.4-pre.13, justanarrowheadmold@1.0.1, norottinghides@1.0.0, noticeboard@1.1.0, oneroof@1.2.3, rivers@3.2.0, simplefootstepsredux@1.0.1, somethinginthewater@1.2.0, spawnersapi@1.1.2, stablesurface@0.0.2, stonebakeoven@1.1.4, storageoptions@1.0.2, th3restock@1.1.2, unconscious@1.7.0, vanvar@6.0.4, creative@1.20.0-rc.1, vsquest@1.2.0, survival@1.20.0-rc.1, vsvillage@1.0.6, battletowers@1.0.5, playercorpse@1.11.0, statushudcont@2.1.2, stonequarry@3.4.1, vsvillageaged@0.0.4, wildcraftfruit@1.2.2, wildcraftherb@0.0.1, wildcrafttree@1.2.0
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.GameContent.BlockEntityMusicTrigger.<StopMusic>b__14_0() in VSSurvivalMod\Lore\ResoArchives\BlockEntityMusicTrigger.cs:line 134
   at Vintagestory.API.Client.MusicTrack.<>c__DisplayClass20_0.<FadeOut>b__0(ILoadedSound sound) in VintagestoryApi\Client\Audio\MusicTrack.cs:line 153
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 670
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 92
   at OpenTK.Windowing.Desktop.GameWindow.Run()
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs) in VintagestoryLib\Client\ClientProgram.cs:line 329
   at Vintagestory.Client.ClientProgram.<>c__DisplayClass10_0.<.ctor>b__1() in VintagestoryLib\Client\ClientProgram.cs:line 130
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93

Event Log entries for Vintagestory.exe, the latest 3
==================================
{ TimeGenerated = 9/12/2024 5:28:50 PM, Site = , Source = Application Error, Message = Faulting application name: Vintagestory.exe, version: 1.20.0.0, time stamp: 0x66470000
Faulting module name: KERNELBASE.dll, version: 10.0.19041.5131, time stamp: 0x011921da
Exception code: 0xe0434352
Fault offset: 0x000000000003b699
Faulting process id: 0x35a0
Faulting application start time: 0x01db4a037ead7c16
Faulting application path: C:\Users\User\AppData\Roaming\Vintagestory_experimental\Vintagestory_stable\Vintagestory.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: 5062c959-7e94-4754-bf3d-3012b28d845d
Faulting package full name: 
Faulting package-relative application ID:  }
--------------
{ TimeGenerated = 9/12/2024 5:27:56 PM, Site = , Source = Application Error, Message = Faulting application name: Vintagestory.exe, version: 1.20.0.0, time stamp: 0x66470000
Faulting module name: KERNELBASE.dll, version: 10.0.19041.5131, time stamp: 0x011921da
Exception code: 0xe0434352
Fault offset: 0x000000000003b699
Faulting process id: 0x21cc
Faulting application start time: 0x01db4a0370792209
Faulting application path: C:\Users\User\AppData\Roaming\Vintagestory_experimental\Vintagestory_stable\Vintagestory.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: fffc4cff-590b-448c-847a-0d826cc126bd
Faulting package full name: 
Faulting package-relative application ID:  }
--------------
{ TimeGenerated = 9/12/2024 5:24:57 PM, Site = , Source = Application Error, Message = Faulting application name: Vintagestory.exe, version: 1.20.0.0, time stamp: 0x66470000
Faulting module name: KERNELBASE.dll, version: 10.0.19041.5131, time stamp: 0x011921da
Exception code: 0xe0434352
Fault offset: 0x000000000003b699
Faulting process id: 0x3a30
Faulting application start time: 0x01db49fe7cdea795
Faulting application path: C:\Users\User\AppData\Roaming\Vintagestory_experimental\Vintagestory_stable\Vintagestory.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: 8f11f5e0-39ab-457f-87fe-cdb4d2249162
Faulting package full name: 
Faulting package-relative application ID:  }```

**Comment by Craluminum2413** [comment]
Duplicate of https://github.com/anegostudios/VintageStory-Issues/issues/4603

**Comment by radfast** [comment]
Thanks for reporting, this will be fixed in upcoming 1.20-rc.2
